services/core-web/src/tests/models/document.spec.ts

```diff
@@ -84,6 +84,7 @@ describe("MineDocument model", () => {
       FileOperations.Replace,
       FileOperations.Archive,
       FileOperations.Delete,
+      FileOperations.MultiSelect,
```

Comment:
Do we actually need a separate permission to multi select? Multi-selecting doesn't technically perform any actions to the file.

Reply:
Hmm... actually I think this is a good point. Technically according to the figma mockup, the permission check is to determine if the user will see a drop down or a download button for the bulk actions, so regardless of permissions the user should be able to bulk download, the only other action in the drop down is bulk archive so we can actually just check for archive permission here.

Reply:
MultiSelect has been removed now and is using document's archive permission instead to check.